Scripts/install.sh

```diff
@@ -1,11 +1,26 @@
 #!/bin/sh
 
+set -euo pipefail
+
 DOWNLOAD_URI=https://github.com/supermarin/Alcatraz/releases/download/1.1.10/Alcatraz.tar.gz
 PLUGINS_DIR="${HOME}/Library/Application Support/Developer/Shared/Xcode/Plug-ins"
+XCODE_VERSION="$(xcrun xcodebuild -version | head -n1 | awk '{ print $2 }')"
+PLIST_PLUGINS_KEY="DVTPlugInManagerNonApplePlugIns-Xcode-${XCODE_VERSION}"
+BUNDLE_ID="com.mneorr.Alcatraz"
+TMP_FILE="/tmp/${BUNDLE_ID}.xcode-defaults"
+
+# Remove Alcatraz from Xcode's skipped plugins list if needed
+defaults read -app Xcode "$PLIST_PLUGINS_KEY" > "$TMP_FILE"
+/usr/libexec/PlistBuddy -c "delete skipped:$BUNDLE_ID" "$TMP_FILE" > /dev/null 2>&1 && {
+	pgrep Xcode > /dev/null && { echo 'An instance of Xcode is currently running. Please close Xcode before installing Alcatraz.'; exit 1; }
```

Comment:
Its easier to read this section if you spread everything after the curly brace onto separate lines, just like the outer enclosure.